src/test/java/com/auth0/client/mgmt/UsersEntityTest.java

```diff
@@ -574,6 +574,20 @@ public void shouldThrowOnLinkUserIdentityWithNullSecondaryId() throws Exception
         api.users().linkIdentity("1", null, "auth0", null);
     }
 
+    @Test
+    public void shouldThrowOnLinkUserIdentityWithNullSecondaryToken() throws Exception {
+        exception.expect(IllegalArgumentException.class);
+        exception.expectMessage("'secondary id token' cannot be null!");
+        api.users().linkIdentity("1", null);
+    }
+
+    @Test
+    public void shouldThrowOnLinkUserIdentityWithNullUserId() throws Exception {
+        exception.expect(IllegalArgumentException.class);
+        exception.expectMessage("'primary user id' cannot be null!");
+        api.users().linkIdentity("1", null);
```

Comment:
this should fix the tests
```suggestion
        api.users().linkIdentity(null, "2");
```